language/Pattern-Matching.md

```diff
@@ -33,6 +33,7 @@ The following pattern types are supported:
 - Record patterns
 - Named patterns
 - Guards
+- Pattern guards
```

Comment:
Pattern guards are not a type of pattern, they're a context in which patterns can be used, so I don't think they should be in this list.